qa/L0_perf_nomodel/run_test.sh

```diff
@@ -42,26 +41,38 @@ TENSOR_SIZE=${TENSOR_SIZE:=1}
 SHARED_MEMORY=${SHARED_MEMORY:="none"}
 REPORTER=../common/reporter.py
 
-DATADIR=/data/inferenceserver/${REPO_VERSION}
+DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
 RESULTDIR=${RESULTDIR:=.}
 
-SERVER=/opt/tritonserver/bin/tritonserver
-SERVER_ARGS="--model-repository=`pwd`/models"
+OPTDIR=${OPTDIR:="/opt"}
```

Comment:
I think you should change this to be "TRITON_DIR" and default to `/opt/tritonserver`, which simply means the directory that is structured in the way you expect as below. Currently it feels to be in the middle, as the user can configure the path to the directory but not the name of the directory as well.

Reply:
Sure, making this change